test/system/035-logs.bats

```diff
@@ -328,12 +330,12 @@ function _log_test_follow_since() {
 
     # sleep is required to make sure the podman event backend no longer sees the start event in the log
     # This value must be greater or equal than the value given in --since below
-    sleep 0.2
+    sleep 0.5
```

Comment:
I don't think this is a proper fix, the issue can go in both directions. If we are slow here and need 1.1 s in total the log line printed in the ctr will be missed and the next line is only written after 5s so I think at the very least we should make the container print every 1s or something